### PR TITLE
Docker2, needs newer NXRM

### PR DESCRIPTION
**Pull request by bhamail**
This is a work-in-progress branch to illustrate a simpler Dockerfile. However, it requires a newer version of Nexus Repository Manager be the parent before it will work.

The most useful file in this branch is the Dockerfile, and maybe some of the README changes.

I just had a simpler thought: I you include the Dockerfile from this PR and some related README changes in your development branch, then feel free to close out this PR. :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# see: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
+ARG NEXUS_VERSION=latest
+
+FROM maven:3-jdk-8-alpine AS build
+
+COPY . /nexus-repository-r/
+RUN cd /nexus-repository-r/; \
+    mvn clean package -PbuildKar;
+
+FROM sonatype/nexus3:$NEXUS_VERSION
```
Comment (aornatovskyy) [inline Dockerfile]: I could be wrong but looks like {} is missed

Comment (bhamail) [reply Dockerfile]: From our session last week, this is working as it, but were you able to get it working on your machine after pruning, etc?

Comment (aornatovskyy) [reply Dockerfile]: Hi @bhamail . Sorry for so long response. Will try to finish my verification today.

Comment (bhamail) [reply Dockerfile]: I couldn't make this fail as is. Did you have any luck verifying it?

Comment (aornatovskyy) [reply Dockerfile]: it works and ARG variable is ok.

**Pull request merged**